catalog/core/catalog-core-transform-impl/src/main/java/org/codice/ddf/catalog/transform/impl/AbstractTransformerAdapter.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.transform.impl;
+
+import ddf.catalog.Constants;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import javax.activation.MimeType;
+import javax.activation.MimeTypeParseException;
+import org.codice.ddf.catalog.transform.TransformerProperties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public abstract class AbstractTransformerAdapter implements TransformerProperties {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(AbstractTransformerAdapter.class);
+
+  private static final String NOT_AVAILABLE = "N/A";
```

Comment:
✏️ is there something that dictates this constant?  I would read this as `Not Applicable`

Reply:
This is how the original code handled the case of a service properties that was missing the id. I don't know if something else is expecting the value to be "N/A".